src/utils/cosmosdb_model.ts

```diff
@@ -212,6 +217,19 @@ export abstract class CosmosdbModel<
     )(newDocument, options);
   }
 
+  public batch(
+    // eslint-disable-next-line
+    operations: OperationInput[],
+    partitionKey?: NonEmptyString,
+    options?: RequestOptions
+    // eslint-disable-next-line
+  ): TE.TaskEither<CosmosErrors, any> {
```

Comment:
The model return always a validated output. In here we return any response. We are only wrapping the SDK method loosing any model logic.
What is the output? Maybe `TE.TaskEither<CosmosErrors, Response<ReadonlyArray<TR>>>`?

Reply:
You are right, fixed it in [c529b4e](https://github.com/pagopa/io-functions-commons/pull/296/commits/c529b4e7961724b6de93d7ed32652c1dd342c3c8)